docs/guides/installation_uv.md

```diff
@@ -0,0 +1,260 @@
+
+# Running Marimo with `uv`
+
+You can run a Marimo notebook using the package manager `uv` through one of the following methods:
+
+1. **Within a project environment**  
+2. **Temporary installation (cached)**  
+3. **Directly from a URL**  
+4. **Globally Installed Tool**
+
+---
+
+## Running in a Project Environment
+
+We recommend running Marimo within a project-specific environment.  
+Here's how you can set it up:
+
+```bash
+uv venv
+uv pip install marimo
+uv run marimo edit hi.py
+```
+
+The `uv run` command provides the simplest way to access the virtual environment.   
+
+However, if you prefer to activate it manually, you can do so with the following commands:
+
+```bash
+source .venv/bin/activate
+marimo edit hi.py
+``` 
+
+---
+### Using Specific Python Versions
+
+To specify a Python version, add the `--python` option while creating the venv. For example:
+
+```bash
+uv venv --python 3.13
+```
+
+```{admonition} Tip
+:class: tip
+
+uv will automatically fetch Python versions as needed — you don't need to install Python to get started.
+```
+<!-- Source: https://docs.astral.sh/uv/guides/install-python/#installing-python -->
+
+
+If no version is specified, the default Python version is used (currently: **[TODO: Specify default Python version]**).
+
+---
+
+### Installing Packages
+
+There are three ways to install additional packages:
+
+1. **Via Terminal**  
+   Run the following command:  
+   ```bash
+   uv pip install matplotlib polars
+   ```
+
+2. **Within the Notebook**  
+   Type `import polars as pd`. Run the cell, and a "Missing package" pop up window will appear.
+
+<div align="center">
+<figure>
+<img src="/_static/image.png" width="650px"/>
+<figcaption>
+Screenshot demonstrating the "Missing package" window.
+</figcaption>
+</figure>
+</div>
+
+
+3. **Using the Packages Tab**  
+   Navigate to the "Packages" tab and select the desired package.
+
+<div align="center">
+<figure>
+<img src="/_static/image-1.png" width="400px"/>
+<figcaption>
+Screenshot demonstrating the "Packages" tab.
+</figcaption>
+</figure>
+</div>
+
+
+### Defining Dependencies with `pyproject.toml`
+
+A pyproject.toml file makes it easier to manage your project’s dependencies in one place. Using `uv`, you can quickly set up and customize your project’s environment
+
+```bash
+uv init           # Creates a pyproject.toml file
+uv add marimo     # Adds "marimo>=0.9.31" to dependencies
+uv run marimo edit hi.py
+```
+
+
+### Reading a `pyproject.toml` from an Existing Project
+
+If you already have a `pyproject.toml` file—for example, when cloning an existing project—you can use the `uv sync` command to synchronize and install the dependencies defined within it:
+
+```bash
+uv sync
+uv run marimo edit hi.py
+
+```
+
+
```

Comment:
not sure what IDE you are using, but most of markdown plugins that will autoformat your markdown. it would be useful to install to avoid PR comments about it

Reply:
Good point! I use the "Prettier" Markdown formatter and will re-format the code.